src/responsibleai/rai_analyse/_score_card/classification_components.py

```diff
@@ -168,10 +168,16 @@ def get_metric_bar_plot(mname, data):
             True if i == data["classes"][1] else False for i in data["y_test"]
         ]
         class_0_metric = get_metric(
-            mname, data["y_pred"][y_0_filtermap], data["y_test"][y_0_filtermap]
+            mname, data["y_test"][y_0_filtermap],
```

Comment:
Ilya had comment: 
can some of the repeating strings be moved to constants, like y_test and classes?